app/controllers/users_controller.rb

```diff
@@ -1,12 +1,12 @@
 class UsersController < ApplicationController
   def index
     @users = User.all
-    render json: @users
+    render 'index'
   end
 
   def show
     @user = User.find(params[:id])
     @user_posts = @user.posts
-    render json: @user_posts
+    render 'show'
```

Comment:
[OPTIONAL]
- [ ] Same here as pointed in the previous comment :point_up_2: rails will render the `app/views/users/show.html.erb` by default for the `show` action in the users controller. Therefore I suggest you remove this `render 'show'`

Reply:
👍